index.html

```diff
@@ -205,6 +205,41 @@ <h2 class="lang">Skills</h2>
             </li>
         </ul>
     </section>
+    <section class="form-section">
+        <ul class="form-iner">
+            <li>
+              <h2 class="form-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            </li>
+            <li>
+              <form action="https://formspree.io/f/xyyoglkw" method="post" class="message-form">
+                <input type="text" name="name" placeholder="Name" maxlength="30" required class="form-control">
+                <input type="email" name="email" placeholder="Email" maxlength="30" required class="form-control">
+                <textarea name="message" placeholder="Enter message!" rows="5" maxlength="500" required class="form-control"></textarea>
+                <button type="submit" class="btn-form ">Get in touch</button>
+              </form>
+            </li>
+          </ul>
+    </section>
+    <footer class="footer-nav">
+        <ul class="icon-list">
+            <li>
+            <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i> </a>
+            </li>
+            <li>
+            <a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i> </a>
+            </li>
+            <li>
+            <a href="#"><i class="fa fa-medium" aria-hidden="true"></i> </a>
+            </li>
+            <li>
+            <a href="#"><i class="fa fa-github" aria-hidden="true"></i> </a>
+            </li>
+            <li>
+            <a href="#"><i class="fa fa-angellist" aria-hidden="true"></i> </a>
+            </li>
+        </ul>
+        <h3 class="email-title"><a href="tobinchilongojnr@gmail.com">tobinchilongojnr@gmail.com</a></h3>
```

Comment:
- [ OPTIONAL] Nice one using the email link. However, please make sure you add your correct email address as the response I am getting when I click on your email address is a 404 page error 😄 .

<img width="240" alt="chrome_fVlXZrioMT" src="https://user-images.githubusercontent.com/81468675/144248307-ac137771-06e1-47e5-8f0d-6d89d5a4df0d.png">